src/main/scala/gov/wicourts/json/formlet/ValidationErrors.scala

```diff
@@ -104,19 +103,19 @@ object ValidationErrors {
 
   def collapse(errors: ValidationErrors): ValidationErrors = {
     def collapseAll[A](l: List[(A, ValidationErrors)]): ValidationErrors =
-      l.map(_._2).map(collapse).suml
+      l.map(_._2).map(collapse).combineAll
 
     dedup(
       errors match {
-        case f@FieldErrors(_) => f
+        case f @ FieldErrors(_) => f
         case ObjectErrors(l) => collapseAll(l)
         case ArrayErrors(l) => collapseAll(l)
-      }
+      },
     )
   }
 
   def string(name: String, error: String): ValidationErrors =
-    ObjectErrors(List((name, FieldErrors(NonEmptyList(error)))))
+    ObjectErrors(List((name, FieldErrors(NonEmptyList.of(error)))))
```

Comment:
Not a big deal, but this should probably be `NonEmptyList.one`.

Reply:
There are quite a few of these in this MR (if you want to throw sed at it).

Reply:
Definitely don't trust myself to do that and *save* effort. I'll get a scalafix recipe for it in the main project